express-stock-trading-platform/server060.js

```diff
@@ -18,11 +18,18 @@ const prices = {
   OPQ: 0.48,
   RST: 9.32,
   UVW: 10.94,
-  XYZ: 5.32,
+  XYZ: 5.32
 };
 
 app.get('/buy/:ticker/:shares', (req, res) => {
-  // 070. Express allows us to access the :ticker route parameter in the callback with the 'params' object, like 'req.params.ticker'.  Create two variables name 'ticker' and 'shares', assign them the corresponding values from 'req'.
   const ticker = req.params.ticker;
   const shares = req.params.shares;
 });
+
+//Now we need determine the total amount of the purchase, which equals to 'shares * price per share'.
```

Comment:
//Now we need calculate the total amount of the purchase.
//You can determine the price per share from the `prices` object we previously declared above.
//For example, if we wanted the price for stock MNO, we would use `prices['MNO']`
//To get the price for the stock we need, we would use `prices[ticker]`
//Create a variable named 'total' and set it equal to `shares * prices[ticker]`.